### ISSUE TITLE
Modify settings from chat command

### ISSUE THREAD

**Issue by pajlada** [Chatterino/chatterino2#2635]
**What should be added?**
A chat command that allows the user to modify most (if not all) normal settings.

**Why should it be added?**
Settings such as streamer mode could want to be toggled on a whim, and a chat command would allow the user to change this faster. (See #2487 and #2489)

The feature should have some sort of autocompletion/popup when used so that setting keys can be autocompleted with a description for the setting.

**Comment by KararTY** [comment]
I like this idea more. Will cover the connected issues fairly well, and I guess since #2632 allows custom commands to run one can make `/dnd %1` commands to cover something like `/setting streamermode %1` fairly easily.

**Comment by Mm2PL** [comment]
requires implementation of #467 for commands.

**Comment by pajlada** [comment]
> requires implementation of #467 for commands.

Not quite, the intent behind #467 was different and was only meant to apply to username and __maybe__ emote completions. However, what we would need in this case, is some way to make autocompletions add 'command autocompletions' if it detects the message that comes before the <TAB> is a command.
